Sources/SmithyJSON/NodeInfo.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public struct NodeInfo: Equatable {
```

Comment:
Like with FormURL, the JSON NodeInfo type holds only name since it has no concept of namespaces, etc. like XML.